jobs/cloud_controller_clock/spec

```diff
@@ -27,6 +27,10 @@ templates:
   db_ca.crt.erb: config/certs/db_ca.crt
   credhub_ca.crt.erb: config/certs/credhub_ca.crt
 
+  opi_tls.crt.erb: config/certs/opi_tls.crt
```

Comment:
I imagine we want these to be in sync across all of the `capi-release` jobs that need it. Can you you expose the properties on the `cloud_controller_internal` link in the `cloud_controller_ng` job and just take the values off of that? It's our super-link that we use for sharing lots of stuff between jobs (a BOSH best practice I'm sure 😅 ).

https://github.com/cloudfoundry/capi-release/blob/12f8865108fc29aeaf6439e95f86a42b23c26a6d/jobs/cloud_controller_ng/spec#L118

Reply:
Resolved with `49ff53cc9dd753a0401bec63d32e8c3ea0575842`